source/installation/index.html

```diff
@@ -6,6 +6,25 @@
 toc: true
 ---
 
+<p>
+Home Assistant can be installed in 4 flavors:
+</p>
+<ul>
+    <li><em>Home Assistant Operating System</em>: Minimal Operating System optimized to power Home Assistant. Comes with Supervisor pre-installed. Recommended for new users.</li>
```

Comment:
```suggestion
    <li><em>Home Assistant Operating System</em>: Minimal Operating System optimized to power Home Assistant. Comes with the Supervisor pre-installed. Recommended for new users.</li>
```

Additionally, is it recommended for new users? As in, this is recommended in general, right?

Reply:
Agreed, recommend it in general make sense. Will change that.